src/lora.cpp

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
See some of the other modules, it would be good to use the MicroTasks framework so the LORA module can manage the timings and listen in on events, etc.